Taskfile.yml

```diff
@@ -117,6 +119,43 @@ tasks:
         # stderr for logging.
         sh: echo '{{.INPUT}}' | ./bin/check 2>&1 1>/dev/null
 
+  test:acceptance:
+    desc: Run the Cogito acceptance tests. Needs a running Concourse.
+    cmds:
+      - task: test:acceptance:set-pipeline
+      - task: test:acceptance:chat-message-file
+      - task: test:acceptance:chat-message-append
+
+  test:acceptance:fly-login:
+    desc: Performs a fly login in the target to be used in the acceptance tests.
+    cmds:
+      - fly -t cogito login -c $(gopass show cogito/concourse_url) --open-browser
+
+  test:acceptance:set-pipeline:
+    desc: Set the acceptance test pipeline
+    cmds:
+      - >
+        fly -t cogito set-pipeline --non-interactive -p cogito-acceptance-{{.BRANCH}}
+        -c pipelines/cogito-acceptance.yml
+        -y target-branch=stable
+        -y cogito-branch={{.BRANCH}}
+        -y github-owner=$(gopass show cogito/test_repo_owner)
+        -y repo-name=$(gopass show cogito/test_repo_name)
+        -y oauth-personal-access-token=$(gopass show cogito/test_oauth_token)
+        -y cogito-tag={{.BRANCH}}
+        -y gchat_webhook=$(gopass show cogito/test_gchat_webhook)
+      - fly -t cogito unpause-pipeline -p cogito-acceptance-{{.BRANCH}}
+
+  test:acceptance:chat-message-file:
+    desc: Run a pipeline job to test chat_message_file
+    cmds:
+      - fly -t cogito tj -j cogito-acceptance-{{.BRANCH}}/chat-message-file.job -w
```

Comment:
I started using the short versions to keep the commands on one YAML (not too long) line, but I have to agree that now it is not uniform. I also have to agree that the short versions are quite cryptic.

Would you be OK if we use the short versions everywhere? Otherwise, I will come up with something uniform and decently readable.

Reply:
Given we set the line length limit for python code to 99 and here it would just reach 91 I think it's not too long:
```
+>>> len("      - fly -t cogito tj -j cogito-acceptance-{{.BRANCH}}/chat-message-file.job -w")
82
+>>> len("      - fly -t cogito trigger-job -j cogito-acceptance-{{.BRANCH}}/chat-message-file.job -w")
91
```

But we're using short version of the option flags anyway so familiarity with fly is already assumed so pushing further and using short version of the commands should be fine too.

Reply:
Done.